src/Device.ts

```diff
@@ -61,10 +62,10 @@ Network.get = singleton(() => new NetworkNode());
 const logger = Logger.get("Device");
 
 class Device {
-    async start() {
+    async start(networkInterface : string|undefined = undefined) {
```

Comment:
You can simply make it optional:
start(networkInterface?: string)

Reply:
i was sleepy. ;) Fixing.